Dockerfile

```diff
@@ -58,16 +59,10 @@ RUN cd /etc/apache2/mods-enabled \
 # Download and extract Known distribution
 RUN mkdir -p /var/www/known \
   && cd /var/www/known \
-  && curl -ko known.zip ${known_git_url} \
-  && unzip known.zip \
+  && curl -s ${known_git_url} | tar xzf - \
   && mv known-master/* /var/www/known \
   && mv known-master/.htaccess /var/www/known \
-  && rm known-master/.gitignore \
-  && rm known-master/.travis.yml \
-  && rm known-master/.babelrc \
-  && rm -r known-master/.github \
-  && rmdir known-master \
-  && rm known.zip
```

Comment:
I added these explicit steps initially to not forget anything new added to the directory. But as we move all non-hidden files in the first place and I currently can't think of any more hidden files than the .htaccess, your option of deleting the whole known-master directory with 'rm -r' is the better choice.